libqpdf/QPDFEFStreamObjectHelper.cc

```diff
@@ -141,7 +141,7 @@ QPDFEFStreamObjectHelper::setModDate(std::string const& date)
 QPDFEFStreamObjectHelper&
 QPDFEFStreamObjectHelper::setSubtype(std::string const& subtype)
 {
-    setParam("/Subtype", QPDFObjectHandle::newName("/" + subtype));
+    this->oh.getDict().replaceKey("/Subtype", QPDFObjectHandle::newName("/" + subtype));
```

Comment:
Everything looks good here assuming CI passes, which it looks like it will. I have a preference for keeping my lines within 80 columns. I would break the line after `replaceKey(` and indent `"/Subtype"...` four spaces in from the previous line for stylistic consistency. See `stream.warnIfPossible` on line 159. Then I will gladly merge this change.

Reply:
Done, thanks!